examples/mecci_kissinger.py

```
@@ -0,0 +1,234 @@
+                    heating_rates.append(heating_rate)
+                    logger.info(f"Processed {filename}: Peak Temperature = {peak_temp:.2f} K, Heating Rate = {heating_rate:.2f} K/min")
+                    plot_dsc_with_derivatives(temp_heating, hf_heating, filename, temp_range, peak_temp, exothermic)
+                else:
+                    logger.warning(f"No peak found in {filename}")
+            except Exception as e:
+                logger.error(f"Error processing {filename}: {str(e)}")
+
+    if peak_temperatures and heating_rates:
+        try:
+            e_a, a, se_e_a, se_ln_a, r_squared = kissinger_method(np.array(peak_temperatures), np.array(heating_rates))
+            logger.info(f"Kissinger analysis results:")
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)